.github/workflows/oci-ocm-multiarch.yaml

```diff
@@ -321,3 +326,12 @@ jobs:
             {resource_str}
             ```
             '''))
+      - uses: actions/checkout@v4
+        with:
+          repository: ccwienk/cc-utils
```

Comment:
leftover? 😄

Reply:
darn. I think I did not push that chunk. thx

Reply:
@zkdev : now I see what happened. I only wanted to push the first commit of those three. please re-review..